src/librustc_target/spec/wasm32_unknown_unknown.rs

```diff
@@ -65,7 +65,6 @@ pub fn target() -> Result<Target, String> {
         target_vendor: "unknown".to_string(),
         data_layout: "e-m:e-p:32:32-i64:64-n32:64-S128".to_string(),
         arch: "wasm32".to_string(),
-        linker_flavor: LinkerFlavor::Lld(LldFlavor::Wasm),
```

Comment:
Similar to emscripten above I think this means that wasm will attempt to use `cc` by default if we don't otherwise set some form of a default here

Reply:
Or maybe it's best to sum this up as "we probably want to continue have default linker flavors", I think?